Userland/Libraries/LibWeb/PixelUnits.h

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright (c) 2022, Sam Atkins <atkinssj@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Concepts.h>
+#include <AK/DistinctNumeric.h>
+#include <AK/Traits.h>
+#include <LibGfx/Forward.h>
+#include <math.h>
+
+namespace Web {
+
+/// DevicePixels: A position or length on the physical display.
+AK_TYPEDEF_DISTINCT_NUMERIC_GENERAL(int, DevicePixels, Arithmetic, CastToUnderlying, Comparison, Increment);
+
+template<Arithmetic T>
+constexpr bool operator==(DevicePixels const& left, T const& right) { return left.value() == right; }
```

Comment:
Nit: All these `DevicePixels const&` or `CSSPixels const&` can just be passed by value instead. 
They are after all just an `int` or `float` wrapped in a class.

Reply:
(all these ones in a header will almost certainly be inlined anyway, but the `const &` is just extra noise :P)

Reply:
Done! :^)